app/payment/session.go

```diff
@@ -114,6 +123,24 @@ func RespondPayChProposal(pctx context.Context, s perun.SessionAPI, proposalID s
 	return s.RespondChProposal(pctx, proposalID, accept)
 }
 
+// SubPayChCloses registers a subscription for payment channel closes.
+func SubPayChCloses(s perun.SessionAPI, notifier PayChCloseNotifier) error {
+	return s.SubChCloses(func(notif perun.ChCloseNotif) {
+		notifier(PayChCloseNotif{
+			ClosingState: PayChInfo{
+				ChannelID: notif.ChannelID,
+				BalInfo:   balsFromState(notif.Currency, notif.ChState, notif.Parts),
+				Version:   fmt.Sprintf("%d", notif.ChState.Version),
+			},
+		})
+	})
+}
+
+// UnsubPayChCloses unregisters a subscription for payment channel closes.
```

Comment:
```suggestion
// UnsubPayChCloses unsubscribe a subscription for payment channel closes.
```

Reply:
Changed to deletes the existing subscription for ...